v-next/hardhat/test/internal/cli/init/init.ts

```diff
@@ -311,6 +312,72 @@ describe("installProjectDependencies", async () => {
       }
     },
   );
+
+  it(
+    "should not update dependencies if they are up-to-date and the user opts-in to the update (specific version)",
+    {
+      skip: process.env.HARDHAT_DISABLE_SLOW_TESTS === "true",
+    },
+    async () => {
+      const template: Template = {
+        name: "test",
+        packageJson: {
+          name: "test",
+          version: "0.0.1",
+          devDependencies: { hardhat: "^3.0.0-next.0" }, // <-- required version
```

Comment:
When reading through it for the first time, I started worrying and double-checking if we were going to have to update the version here with every release. Now I know we don't, but I guess we could still consider using clearly fake dependency to avoid that momentary confusion for others in the future. Something like:

```suggestion
          devDependencies: { fake-dependency: "^1.2.3-dev" }, // <-- required version
```

It's all correct so I'm not requesting this change definitively. Just putting it out there to consider if you want to.